item-spec/item-spec.md

```diff
@@ -1,6 +1,10 @@
-# STAC Item Specification
+# STAC Item and ItemCollection Specification
 
-This document explains the structure and content of a SpatioTemporal Asset Catalog (STAC) Item. Each
+This document explains the structure and content of a SpatioTemporal Asset Catalog (STAC) Item and ItemCollection. 
+
+## Item fields
```

Comment:
This section should be named just "Item" as it explains more then just the Item fields.